src/docs/Contributing/NewPattern/SubmitNewPattern/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Submit a new pattern'
+order: 10
+---
+
+The creation of a new pattern on MOZAIC is usually done via the following workflow:
+
+## 1. Create a new branch
+
+Creating your new pattern starts with creating a github branch dedicated to it.  
+This branch is ideally named as follows: `pattern-xxx`.
+
+**xxx** refers to the name of your pattern _(exemple : **pattern-flags** ou **pattern-ratingstars**)_.
+
+
+## 2. Create your folder structure using Plop
+
+Plop is a simple generator.  
+We use it to save bootstrapping time when creating new pattern.
+
+To use it, simply type in your terminal `npx plop pattern`
+
+Then the CLI will ask you 3 questions : 
+
+- the pattern name : use a Capital first letter and no spaces (ex: Buttons)
+- the pattern path : from the `src/docs` directory with no `/` at the end (ex: Components)
+- the pattern order : the position of your pattern in the navigation menu (an integer - ex: 4)
+
+At the end of the process Plop will generate the required `usage`, `code`, `design`, `specs` tabs with helping markdown content.
```

Comment:
You must reverse design and specs, to have the same order that we have on the site : 1. usage, 2. code, 3. specs and 4. design

Reply:
It's fixed and I added a link at the top of the page to redirect designers to definition of done.